src/collections/members/usman-siddique/index.mdx

```diff
@@ -0,0 +1,11 @@
+---
+name: Usman Siddique
+position: Contributor
+image_path: ./Usman-Dev.jpg
+github: UsmanDev09 
+linkedin: muhammadusmansiddique
+status: Active
+location: Lahore, Punjab, Pakistan
```

Comment:
It should be Lahore, Pakistan, right @UsmanDev09 ?

Reply:
Yes, correct

Reply:
sure